.github/workflows/build_and_upload_ffmpeg.yaml

```
@@ -0,0 +1,38 @@
+name: 'build & upload ffmpeg artifacts'
+
+on:
+ schedule:
+ - cron: 0 0 * * 1 # Every Monday at 12:00 AM UTC
+ workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v6
+
+      - name: install dependencies
+        run: |
+          sudo apt update
+          sudo apt upgrade -y
+          sudo apt install build-essential subversion ragel curl texinfo g++ ed bison flex cvs yasm automake libtool autoconf gcc cmake git make pkg-config zlib1g-dev unzip pax nasm gperf autogen bzip2 autoconf-archive p7zip-full meson clang python-is-python3 -y
+      - name: build (Win64)
+        run: echo 3 | ./cross_compile_ffmpeg.sh
+        
+      - name: build (Linux)
+        run: echo 4 | ./cross_compiled_ffmpeg.sh
+      
+      - name: Create Release
+        uses: ncipollo/release-action@v1
+        with: 
+          tag: "auto-${{ github.run_number }}"
+          name: "Auto Build ${{ github.run_number }}"
+          draft: false
+          prerelease: false
```

Use current date instead of run number for release name and tag
```suggestion
      - name: Get date
        id: date
        run: echo "CURRENT_DATE=$(date +%Y-%m-%d)" >> "$GITHUB_OUTPUT"
      - name: Create Release
        uses: ncipollo/release-action@v1
        with: 
          tag: "build-${{ steps.date.outputs.CURRENT_DATE }}"
          name: "Build ${{ steps.date.outputs.CURRENT_DATE }}"
          draft: false
          prerelease: false
```